database/seed.js

```diff
@@ -0,0 +1,180 @@
+  "Toronto, Canada",
+  "Washington DC, United States",
+  "Zurich, Switzerland",
+  "Hong Kong, Hong Kong",
+  "Beijing, China",
+  "Berlin, Germany",
+  "Sydney, Australia",
+  "Las Vegas, United States",
+  "Frankfurt, Germany",
+  "Miami,	United States",
+  "San Diego,	United States",
+  "Seoul,	South Korea",
+  "Prague, Czech Republic",
+  "Munich, Germany",
+  "Houston, United States",
+  "Milan, Italy",
+  "Dublin, Ireland",
+  "Seattle, United States",
+  "Dallas, United States",
+  "Istanbul, Turkey",
+  "Vancouver, Canada",
+  "Melbourne, Australia",
+  "Abu Dhabi, United Arab Emirates",
+  "Vienna, Austria",
+  "Calgary, Canada",
+  "Brussels, Belgium",
+  "Denver, United States",
+  "Doha, Qatar",
+  "Oslo, Norway",
+  "Orlando, United States",
+  "Austin, United States",
+  "Stockholm, Sweden",
+  "Montreal, Canada",
+  "Philadelphia, United States",
+  "Brisbane, Australia",
+  "Atlanta, United States",
+  "Copenhagen, Denmark",
+  "Saint Petersburg, Russia",
+  "Perth, Australia",
+  "Minneapolis, United States",
+  "Lisbon, Portugal",
+  "Venice, Italy",
+  "Portland, United States",
+  "Hamburg, Germany",
+  "Tel Aviv, Israel",
+  "Lyon, France",
+  "Florence, Italy",
+  "Stuttgart, Germany",
+  "Luxembourg, Luxembourg",
+  "Edmonton, Canada",
+  "Osaka, Japan",
+  "Auckland, New Zealand",
+  "Ottawa, Canada",
+  "Budapest, Hungary",
+  "Helsinki, Finland",
+  "Athens, Greece",
+  "Cologne, Germany",
+  "Bangkok, Thailand",
+  "Charlotte, United States",
+  "Phoenix, United States",
+  "New Orleans, United States",
+  "Baltimore, United States",
+  "Valencia, Spain",
+  "Manchester, United Kingdom",
+  "Nashville, United States",
+  "Salt Lake City, United States",
+  "Dusseldorf, Germany",
+  "Sao Paulo, Brazil",
+  "Rio de Janeiro, Brazil",
+  "Raleigh, United States",
+  "Warsaw, Poland",
+  "Marseille, France",
+  "San Antonio, United States",
+  "Birmingham, United Kingdom",
+  "Columbus, United States",
+  "Shanghai, China",
+  "St. Louis, United States",
+  "Detroit, United States",
+  "Sacramento, United States",
+  "Milwaukee, United States",
+  "Kansas City, United States",
+  "Tampa, United States",
+  "Nuremberg, Germany",
+  "Bristol, United Kingdom"
+]
+
+var titles = [];
+
+var split = cities.map((city) => {
+  return city.split(', ');
+});
+
+for (var i = 0; i < 100; i++) {
+  var title = "";
+  title += adjectives[Math.floor(Math.random() * 14)] + " " +
+    nouns[Math.floor(Math.random() * 9)] + " " +
+    prepositions[Math.floor(Math.random() * 2)] + " " +
+    split[i][0];
+  titles.push(title);
+}
+
+var trips = [];
+
+for (var j = 0; j < 100; j++) {
+  var obj = {};
+  obj.title = titles[j];
+  obj.city = cities[j];
+  obj.msrp = ((Math.random() * (2500 - 1500)) + 1500).toFixed(2);
+  obj.price = (obj.msrp * (Math.random() * (.9 - .7) + .7)).toFixed(2);
+  obj.days = Math.floor(Math.random() * (15 - 5)) + 5;
+  obj.dates = [new Date(2020, 1, Math.floor(Math.random() * (30 - 1)) + 1)];
+  trips.push(obj);
```

Comment:
do you want to add an 'id' property as well, 1-100 for each trip?  Jonathan had said we should, so I did.

Reply:
sure i can do that